main.go

```diff
@@ -25,5 +27,7 @@ func main() {
 	router.GET("/request/:requestID", getRequestByID)
 	router.POST("/request", postRequest)
 
-	router.Run("localhost:8080")
+	if err := router.Run("localhost:8080"); err != nil {
```

Comment:
At some point we should add capture of SIGTERM, but that can hold off for a bit.